src/Differ/MapDiffer.php

```diff
@@ -157,6 +157,8 @@ private function getDiffForArrays( array $old, array $new ) {
 	/**
 	 * Returns if an array is associative or not.
 	 *
+	 * @SuppressWarnings(PHPMD.UnusedLocalVariable)
+	 *
```

Comment:
Oh neat! I did not realize this was already fixed and was thinking about submitting a patch myself. When running this rule against some small chunk of fundraising code (~5kloc), we ran into this dozens of times. Which means that for most component the whole rule can currently not be enabled, which is too bad.